backend/src/routers/borrowRequestRouter.ts

```diff
@@ -170,10 +171,17 @@ borrowRequestRouter.post(
  *
  * @apiUse ZodError
  */
-borrowRequestRouter.get('/list', async (_, res) => {
-	const requests = await getActiveBorrowRequests();
-	res.status(200).json(await formatBorrowRequests(requests));
-});
+borrowRequestRouter.get(
+	'/list',
+ isAuthenticated,
+	async (req, res) => {
+  const user = await getAccountFromCid((req.user as GammaUser).cid);
+  if(!user) return;
+		const requests = await getActiveBorrowRequests();
```

Comment:
We need to pass the user
```suggestion
		const requests = await getActiveBorrowRequests(user);
```